drivers: ieee802154: nrf5: load EUI64 from UICR

edmont opened the pull request:
Add `IEEE802154_UICR_EUI64_ENABLE` option to allow loading EUI64 from UICR registers.

Signed-off-by: Eduardo Montoya <eduardo.montoya@nordicsemi.no>

jukkar reviewed (approved):
LGTM

tbursztyka commented on drivers/ieee802154/Kconfig.nrf5:
```diff
@@ -52,4 +52,28 @@ config IEEE802154_NRF5_PKT_TXTIME
 	  Enable TX time support. This is needed when upper layers want
 	  to set the exact time when the packet should be sent.
 
+config IEEE802154_UICR_EUI64_ENABLE
```
s/IEEE802154_UICR_EUI64_ENABLE/IEEE802154_NRF5_UICR_EUI64_ENABLE

tbursztyka reviewed (changes requested):
small option name change requested

edmont committed:
drivers: ieee802154: nrf5: load EUI64 from UICR

Add `IEEE802154_NRF5_UICR_EUI64_ENABLE` option to allow loading EUI64
from UICR registers.

Signed-off-by: Eduardo Montoya <eduardo.montoya@nordicsemi.no>

edmont commented:
@tbursztyka the requested change was applied. Could you verify? Thanks.
